core/drand_test.go

```diff
@@ -337,9 +337,9 @@ func (d *DrandTest2) StartDrand(id string, catchup bool, newGroup bool) {
 	dr := node.drand
 	// we load it from scratch as if the operator restarted its node
 	newDrand, err := LoadDrand(dr.store, dr.opts)
+	//newDrand.opts.clock = node.clock
```

Comment:
This is commented now were the tests failing?

Reply:
yep. it causes a race between a routine reading the clock that spins out from `LoadDrand`. it seems not needed, so :shrug:

Reply:
Gotcha, so, I'd either remove this line if it's really not needed, or if you don't know and don't have the time to look into it then open an issue to get back to fixing this _and_ add a comment here explaining why it's commented out.

Reply:
pushed to remove the line fully

Reply:
That's correct - I can confirm it's not needed - the `dr.opts` comes from the previous node that was here before, so the clock is the same so there is no need to set it explicitely.